src/Jira/Api.php

```diff
@@ -173,14 +173,14 @@ protected function clearLocalCaches()
 	/**
 	 * Get fields definitions.
 	 *
-	 * @return array
+	 * @return array|false
```

Comment:
It's better to wait some time and merge PR with a tests, then merge PR quickly, but without any tests.